### PR TITLE
Add to cart and view cart closes #78

### LINKED ISSUE #78: enable adding products to cart and hook up to cart view (500)

### PR DESCRIPTION
**Pull request by meliis**
on katie's behalf

### CODE REVIEW

File: client/app/app.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ angular.module('stackStoreApp', [
   'ngResource',
   'ngSanitize',
   'ngRoute',
-  'ui.bootstrap'
+  'ui.bootstrap',
```
Comment (meliis) [inline client/app/app.js:8]: aw wish i'd known you were installing ui.bootstrap

File: client/app/cart/cart.controller.js [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
+    $scope.populatedCart;
 
-    $scope.cart = [
-    {name: 'Time Machine', price: 100000, description: {blurb: "A time machine", full:"More than the last"}, qty: 1 },
-    {name: 'Watch', price: 10, description: {blurb: "A time machine", full:"More than the last"}, qty: 1},
-    {name: 'Clock', price: 5, description: {blurb: "A time machine", full:"More than the last"}, qty: 1},
-    ];
-
-    $scope.calculateTotal = function(){
+// Might delete this function (trying to work from cart factory)
```
Comment (Meliis) [inline client/app/cart/cart.controller.js]: yeah this could be done through something like the virtual chuck wrote for Order

**Comment by ChuckPierce** [comment]
This is definitely a lot work.  I am good with it.  Don't really understand the listeners that you have but you can explain it to me later.

**Comment by meliis** [comment]
it's not working for me :(

**Comment by meliis** [comment]
oh--never mind, didn't realize i was still logged in as test user.  yeah i'm not really sure quite how the listeners are working--you add one every time you add a product?

**Comment by wawaontheloose** [comment]
Good to me. I'm a little confused about $scope.cart and the Cart service and why there is both. +1 on listeners too. I guess we will all find out tomorrow!

**Pull request merged**